scripts/automated-translations.ps1

```diff
@@ -135,7 +147,7 @@ else
   if ($changesFromLts)
   {
     Write-Output "`n::group::Pull request`n"
-    $prForChanges = gh pr list --json title,url,headRefName --jq ".[] | select(.headRefName == `"${LtsBranchName}`")"
+    $prForChanges = gh pr list --json title,url,headRefName --jq ".[] | select(.headRefName == `"${TranslationBranchName}`")"
```

Comment:
This was a major issue: it was looking for a PR from the LTS branch instead of from the translation branch.